src/App.jsx

```diff
@@ -94,6 +95,13 @@ const App = ({ children, location, navigate }) => {
 
   useFathom()
 
+  // global styling for the site
+  const GlobalStyle = createGlobalStyle`
```

Comment:
Theres a few concerns with this fix:

* using `createGlobalStyle` affects everything on the page, not just the component, this fix is specific to the `ListMap` component so a fix should stay contained per instance of the component.
* `createGlobalStyle` adds duplicate entries 
![image](https://user-images.githubusercontent.com/649148/196450946-a8b1a3c4-d582-42b1-ac9f-d0cb2452f0cf.png)
* we don't know what side effects changing the properties of the class `.gm-style-moc` might have elsewhere, either on other map instances or within this component.
* it uses `!important` - `!important` is your last resort but if you do need to use it its a good idea to put it in a file called `shame.scss` or `trumps.scss` or `important.scss` so anyone debugging other issues can easily rule out fixes such as these for causing their issues.

Looking at the code the css is applied inline so it would be safer to find and remove the offending line where we know its causing a problem:

```html
<div class="gm-style-moc" style="z-index: 4;position: absolute;height: 100%;width: 100%;padding: 0px;border-width: 0px;margin: 0px;left: 0px;top: 0px;opacity: 0;transition-duration: 0.8s;"><p class="gm-style-mot">Use ⌘ + scroll to zoom the map</p></div>
```

However it looks like its related to this issue:

https://stackoverflow.com/questions/69027683/react-google-maps-api-googlemap-onclick-not-working#comment127169521_69031991
and this issue:
https://github.com/JustFly1984/react-google-maps-api/issues/2906 

so a better fix might be in  `/src/contexts/googleContext.js` on line 15 - change `true` to `false`

```javascript
preventGoogleFontsLoading: false,

Reply:
Will change that right now and push the changes, thanks!